fe1-web/src/core/network/NetworkManager.ts

```diff
@@ -168,3 +168,5 @@ export function getNetworkManager(): NetworkManager {
   }
   return NETWORK_MANAGER_INSTANCE;
 }
+
+export const TEST_ONLY_EXPORTS = { NetworkManager };
```

Comment:
ah, now I understand where the `TEST_ONLY_EXPORTS` comes from. 👍

Reply:
The problem is that without this export I cannot really mock anything as I do not have access to the `NetworkManager` class. People also seem to use [rewire](https://github.com/jhnns/rewire) for this but having a test only export seems to be suggested [here](https://stackoverflow.com/q/54116070/2897827).

Reply:
yeah I like your approach better - much simpler to understand for a new developer on the codebase.